Drawsana/Helpers/DrawsanaUtilities.swift

```diff
@@ -26,8 +26,8 @@ class DrawsanaUtilities {
   }
 
   /// Render an image using CoreGraphics
-  class func renderImage(size: CGSize, _ code: (CGContext) -> Void) -> UIImage? {
-    UIGraphicsBeginImageContextWithOptions(size, false, 0)
+    class func renderImage(size: CGSize, scale:CGFloat = 1.0, _ code: (CGContext) -> Void) -> UIImage? {
+    UIGraphicsBeginImageContextWithOptions(size, false, scale)
```

Comment:
I think this new default value will actually cause the drawing to become non-retina, which is not good. This default should be zero. See `DrawsanaView.redrawAbsolutelyEverything` and `DrawsanaView.drawingDidAddShape`.

Reply:
Hi Steve. You are right, I just changed the parameter to have a default of zero and added couple of comments. However, there is an important distinction of the intention of the renderImage method: If you want to display the image on screen then using the default zero value is good as it will render in the native resolution of the device. However, if you want to save the rendered image to disk you should use 1.0 as the scale value, otherwise the resulting image will be 2x or 3x the original image. Hopefully this is clear. Let me know if you want me to add a few more notes in the documentation about this or make other changes. Cheers!